### PR TITLE
feat: create sign-in page 

### PR DESCRIPTION
**Pull request by jaydelim**
# Description
Created a sign-In page. This page is re-using the `NavBar` component. This page also includes a couple of user-input components: `Email` and `Password` (re-used both components that were previously created from the sign-up page'

# Instruction
Run `npm start`
Visit `http://localhost:3000/`

# Screenshots
## Desktop
Screen Shot 2022-08-08 at 9 35 09 PM

## Tablet
Screen Shot 2022-08-08 at 9 35 02 PM

## Mobile
Screen Shot 2022-08-08 at 9 34 37 PM

### CODE REVIEW

File: src/components/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import InputForm from "./Form";
+
+function SignIn () {
```
Comment (YoussefLaunchUp) [inline src/components/SignIn.js]: This looks pretty similar to the signup page, why don't you make them both into a single component called something like "Auth form" and have them take parameters for what they need.

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]

File: src/pages/SignUpPage.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 import NavBar from '../components/NavBar';
-import SignUp from '../components/SignUp';
+import AuthForm from '../components/AuthForm';
 
 function SignUpPage () {
   return (
     <div>
       <NavBar />
-      <SignUp />
+      <AuthForm
```
Comment (jaydelim) [inline src/pages/SignUpPage.js:9]: `SignUp` and `SignIn` components were using too many share-able codes. They have been deleted and `AuthForm` is now being directly called in `SignUpPage` as well as `SignInPage` for efficiency & simplicity

**Review by YoussefLaunchUp — APPROVED** [review]
Great work!

File: src/components/SignIn.js (hunk 1 above)
Comment (jaydelim) [reply src/components/SignIn.js]: Addressed. Please review my updated codes. Thanks!

**Pull request merged**